<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="midLineWidth">
       <number>2</number>
      </property>
      <widget class="QFrame" name="ThermFrame">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>140</y>
         <width>183</width>
         <height>373</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <widget class="Thermometer" name="Therm" native="true">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>171</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="Bar_frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>243</width>
         <height>243</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <widget class="Barometer" name="Barom" native="true">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>231</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="mainqw_frame">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>150</y>
         <width>483</width>
         <height>363</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <widget class="MainQW" name="Label_pic_main" native="true">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>6</y>
          <width>471</width>
          <height>351</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="DataFrame">
       <property name="geometry">
        <rect>
         <x>249</x>
         <y>67</y>
         <width>481</width>
         <height>81</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Temp. [&lt;span style=&quot; vertical-align:super;&quot;&gt;o&lt;/span&gt;C]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Pressure [hPa]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Rain Int. [%]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>0</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Light Int. [%]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="LabelL">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>40</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="LabelR">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="LabelP">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="LabelT">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>37</x>
         <y>47</y>
         <width>210</width>
         <height>130</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="midLineWidth">
        <number>4</number>
       </property>
       <widget class="DigitalClock" name="Clock" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>192</width>
          <height>112</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhTime</set>
        </property>
       </widget>
      </widget>
      <zorder>ThermFrame</zorder>
      <zorder>Bar_frame</zorder>
      <zorder>mainqw_frame</zorder>
      <zorder>frame</zorder>
      <zorder>DataFrame</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionPo_cz"/>
   <addaction name="actionWykresy"/>
   <addaction name="separator"/>
   <addaction name="actionWyjd"/>
  </widget>
  <action name="actionPo_cz">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/img/connect.png</normaloff>:/new/prefix1/img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Połącz</string>
   </property>
   <property name="toolTip">
    <string>Pokaż okno połącz</string>
   </property>
  </action>
  <action name="actionWykresy">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/img/chart.png</normaloff>
     <normalon>img/chart.png</normalon>:/new/prefix1/img/chart.png</iconset>
   </property>
   <property name="text">
    <string>Wykresy</string>
   </property>
   <property name="toolTip">
    <string>Wyświetl wykresy</string>
   </property>
  </action>
  <action name="actionWyjd">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/new/prefix1/img/exit.png</normaloff>
     <normalon>img/exit.png</normalon>:/new/prefix1/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QWidget</extends>
   <header>digitalclock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainQW</class>
   <extends>QWidget</extends>
   <header location="global">mainqw.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Thermometer</class>
   <extends>QWidget</extends>
   <header location="global">thermometer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Barometer</class>
   <extends>QWidget</extends>
   <header>barometer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
